processing/spikesSiliconProbesV2.m

```diff
@@ -0,0 +1,20 @@
+function [channels, artifacts] = spikesSiliconProbesV2(sourceFile, spikesFile)
+% Spike detection callback for silicon probes.
+% AE 2011-10-26
+
+
+channels = 1 : 32;
+
+% Extract the common reference - mean across channels at each timepoint
+extract_common_referenceSP(sourceFile);
```

Comment:
Ok I updated those files and pushed the changes. 
extract_common_reference is updated to reflect *SP but that's in the ephys-preprocessing repository, and the spikesCb functions for Utah and SiliconProbesV2 both pass the number of channels as an argument